demo/specs/quickstart/gpu-test-mig.yaml

```diff
@@ -0,0 +1,57 @@
+# Run as a deployment with 4 replicas
+# Each replica shares a mig-enabled GPU with a profile of '1g.5gb'.
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gpu-test-mig
+
+---
+apiVersion: resource.k8s.io/v1alpha3
+kind: ResourceClaimTemplate
+metadata:
+  namespace: gpu-test-mig
+  name: mig-devices
+spec:
+  spec:
+    devices:
+      requests:
+      - name: mig-1g-5gb
+        deviceClassName: mig.nvidia.com
+        selectors:
+        - cel:
+            expression: "device.attributes['gpu.nvidia.com'].profile == '1g.5gb'"
+      constraints:
+      - requests: []
+        matchAttribute: "gpu.nvidia.com/parentUUID"
```

Comment:
There is no need for this with just a single MIG device request -- this pulls together the different requests and ensures the allocations come from the same underlying GPU